multicompany_property_product/models/res_partner.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2023 ForgeFlow, S.L.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+
+from odoo import fields, models
+
+
+class PartnerProperty(models.TransientModel):
+    _inherit = "res.partner.property"
+
+    parent_id = fields.Many2one(related="partner_id.parent_id")
+    property_product_pricelist = fields.Many2one(
```

Comment:
I think you don't need to do this kind of thing, as thins kind of thing is done only for company_dependent fields.

What you need to do is only call the method _compute_product_pricelist (which uses force_company) with super and make the properly fix in that method.

Reply:
@MiquelRForgeFlow the problem is that, although it is not technically a company dependent field, logically it is as it ends up being managed as a property. Users that use these multicompany modules expect to handle all company dependent fields via the properties as it helps them see the values of the different companies and update them.

As you correctly say the field is handled via the compute method, that is why I don't get/write the value of the pricelist in the same way as we do for other fields in the properties.

Reply:
Ok.